aptos-move/mvhashmap/src/versioned_aggregators.rs

```diff
@@ -376,6 +378,43 @@ impl<K: Eq + Hash + Clone + Debug + Copy> VersionedAggregators<K> {
         );
     }
 
+    /// Must be called when a snapshot (delta or derived) creation with a given ID
+    /// and initial apply is observed.
+    /// This should be only called when apply applies on top of different ID.
+    pub fn initialize_dependent_delayed_field(
+        &self,
+        id: K,
+        txn_idx: TxnIndex,
+        apply: AggregatorApplyChange<K>,
+    ) {
+        let mut created = VersionedValue::new(None);
+        created.insert(txn_idx, VersionEntry::Apply(apply));
+
+        assert_none!(
+            self.values.insert(id, created),
+            "VerionedValue when creating aggregator ID may not already exist"
```

Comment:
typo here and in initialize_delayed_field: Verioned -> Versioned

Does it make sense for this to be generic or a marco, because initialize delayed field is the same except the entry we are adding? but it's also a few lines, so maybe this way it's more readable.

Reply:
I mean I could've just called a generic insert, but I want to be more explicit , and encapsulate VersionEntry internals to be inside the class